kytos/web-ui/static/js/kytos-logs.js

```diff
@@ -1,84 +1,74 @@
-;(function() {
-  console.log("kytos-logs started")
+var socket = io();
+var channels = new Set();
 
-  var connected=false;
-  var current_line = 0;
-  var socket = io.connect('/logs');
-  // prevent browser from crashing with too many lines
+;(function() {
   var max_lines = 50;
+  var ws_led = $('.nav.sidebar .websocket-status');
 
-  socket.on('connect', function(){
-    turn_on_led()
-    connected = true;
-    console.log('connected')
-    request_log_changes()
-  })
+  socket.on('connect', connect);
+  socket.on('disconnect', disconnect);
+  socket.on('show logs', update_log);
 
-  socket.on('start logs', function(data){
-    if ($('#enable_log')[0].checked)
-    {
-      received_msgs = data.buff
-      current_line = data.last_line
-
-      $.each(received_msgs, function(index, msg){
-        add_log_message(msg, 'controller')
-      });
+  function connect() {
+    console.log('Socket connected.');
+    // restore channel subscriptions
+    for (let channel of channels) {
+      join_channel(channel, true);
     }
-  })
-
-  socket.on('connect_error',function(){
-    connected = false;
-    turn_off_led()
-    console.log('error');
-  })
+    ws_led.addClass('status-online')
+          .removeClass('status-offline');
+  }
 
-  socket.on('disconnect', function(data) {
-    connected = false;
-    turn_off_led()
-    console.log('disconnected')
-  })
+  function disconnect() {
```

Comment:
@cemsbr may be it is better to call this method something like `announce_disconnection` ou `disconnected`, since it does not perform the disconnection itself